helm/prometheus-rules/templates/alerting-rules/app.rules.yml

```diff
@@ -27,11 +27,27 @@ spec:
         sig: none
         team: atlas
         topic: releng
+    - alert: ManagementClusterAppFailedCabbage
+      annotations:
+        description: '{{`Management Cluster App {{ $labels.name }}, version {{ $labels.version }} is {{if $labels.status }} in {{ $labels.status }} state. {{else}} not installed. {{end}}`}}'
+        opsrecipe: app-failed/
+      expr: app_operator_app_info{status!~"(?i:(deployed|cordoned))", catalog=~"control-plane-.*",team=~"cabbage"}
+      for: 30m
+      labels:
+        area: managedservices
+        cancel_if_cluster_status_creating: "true"
+        cancel_if_cluster_status_deleting: "true"
+        cancel_if_cluster_status_updating: "true"
+        cancel_if_outside_working_hours: "true"
+        severity: page
+        sig: none
+        team: cabbage
+        topic: releng
     - alert: ManagementClusterAppFailedHoneybadger
       annotations:
         description: '{{`Management Cluster App {{ $labels.name }}, version {{ $labels.version }} is {{if $labels.status }} in {{ $labels.status }} state. {{else}} not installed. {{end}}`}}'
         opsrecipe: app-failed/
-      expr: app_operator_app_info{status!~"(?i:(deployed|cordoned))", catalog=~"control-plane-.*",team=~"batman|biscuit|halo|honeybadger"}
+      expr: app_operator_app_info{status!~"(?i:(deployed|cordoned))", catalog=~"control-plane-.*",team=~"honeybadger"}
```

Comment:
Hey Mr @JosephSalisbury this is the last PR for the app alert routing 🎉 

For apps in collections the Chart.yaml annotations were updated in https://github.com/giantswarm/giantswarm/issues/19760

As a fallback there is also a mapping in app-exporter from old teams to new teams.

https://github.com/giantswarm/app-exporter/blob/a401deb02d6d3cb2d23233bcfdaabe2728d5596a/helm/app-exporter/templates/configmap.yaml#L20-L25